packages/core/src/server/generator/schema.ts

```diff
@@ -55,23 +53,25 @@ export function getThirdPartyExtensionsTypeDefs() {
   return getTypeDefsFromFolder('thirdParty')
 }
 
-export const nativeApiSchema = getSchema(apiOptions)
```

Comment:
Removes the export of native schema from @faststore/core (that uses functions from faststore/api). Looks that we are not using it anymore and it's directly bound the root cause of some errors.